### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by rayjanwilson** [AtomLinter/linter-jshint#88]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:810:9)
Package.hasActivationCommands (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:735:20)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:189:24)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:167:15)
```

**Comment by avdg** [comment]
The issues below are still open but should be fixed by 619308ebd4d3c060471d28ff6a0275b97c7ea40e, which is released in 0.1.3:

https://github.com/AtomLinter/linter-jshint/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+Package.getActivationCommands+OR+Package.activateConfig

They can all be closed as they are fixed in pr #71

**Comment by Arcanemagus** [comment]
Duplicate of #60.
